modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStoreFactory.java

```diff
@@ -567,4 +593,24 @@ protected String getJDBCUrl(Map params) throws IOException {
         }
         return url;
     }
+
+    /**
+     * Looks up the callback factory with the specified name.
+     * <p>
+     *   Will return {@link JDBCCallbackFactory#NULL} if the factory with the specified name can't be found.
+     * </p>
+     */
+    static JDBCCallbackFactory findCallbackFactory(String factoryName) {
```

Comment:
Not sure what you mean. I did add a static FactoryRegistry instance for looking up instances of the factory. This method uses it.